tests/systems/ippo_system_evaluation_test.py

```diff
@@ -29,6 +29,9 @@ def test_ippo_system_mt() -> System:
     return ippo_system_multi_thread_eval()
 
 
+# TODO Re-add once issue is solved
+# https://github.com/instadeepai/Mava/issues/842
+@pytest.mark.skip
```

Comment:
Can we add this test back in if we merge Omayma's PR?

Reply:
I believe so.